src/data/ACTIONS/root/MNK.ts

```diff
@@ -81,14 +80,15 @@ export const MNK = ensureActions({
 		icon: 'https://xivapi.com/i/002000/002536.png',
 		onGcd: true,
 		speedAttribute: Attribute.SKILL_SPEED,
+		statusesApplied: ['FORMLESS_FIST'],
 	},
 
 	MEDITATION: {
 		id: 3546,
 		name: 'Meditation',
 		icon: 'https://xivapi.com/i/002000/002534.png',
 		onGcd: true,
-		cooldown: 1.2,
+		cooldown: 1200,
```

Comment:
Isn't this a 1s cooldown? I'm probably missing some monk-specific knowledge here.

Reply:
Indeed it is.